src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const whatAction = action.type;
+
+    switch (whatAction) {
```

Comment:
switch/case should always have default case for error handling.
from checklist ⬆️